### ISSUE TITLE
The Colony contract version is getting set incorrectly in the database

### ISSUE THREAD

**Issue by iamsamgibbs** [JoinColony/colonyCDapp#2088]
## Steps to reproduce

Create a new action to upgrade the colony version.

## Expected behaviour

The 'New version' should match the latest available version (14 at the time of writing)

## Actual behaviour

The 'New version' is showing the incorrect version (4)

## Suggestions

It is most likely that the `setCurrentVersion` lambda is not correctly setting the version in the first place.

Running the `getCurrentVersionByKey` query with the key "colony" is returning version 4.

`const currentNetworkVersion = await colonyNetwork.getCurrentColonyVersion();` in the create-data.js script is showing the right version.

**Comment by rdig** [comment]
This local, right?

The version in the db gets overwritten by any event that updates versions (colony or extensions), so the first place to look is whether something else overwrites it, during the creation process

**Comment by iamsamgibbs** [comment]
Yeah this is local. It's showing as version 4 on a fresh environment before running the create-data script.

**Comment by rdig** [comment]
I have a hunch this just requires a colonyJS update, to support the new versions of Colony and Extensions

I think when the block-ingestor runs the seedDb() call it tries to find the version, and since colonyJS doesn't support the versions, falls back to defaults

**Comment by iamsamgibbs** [comment]
Seems like this may only be occurring on branches related to `feat/advanced-payments-2`. I switched between a few master based branches today which were all showing the correct colony version, now after switching to an advanced payments one it is showing incorrectly.
